src/pe_reports/stakeholder/views.py

```diff
@@ -239,14 +234,14 @@ def setCustSubDomain(subdomain, rootUUID, rootname):
             return True
 
     except (Exception, psycopg2.DatabaseError) as err:
-        logging.error("There was a problem logging into the psycopg database %s", err)
+        LOGGER.error("There was a problem logger into the psycopg database %s", err)
```

Comment:
Overzealous search/replace correction:
```suggestion
        LOGGER.error("There was a problem logging into the psycopg database %s", err)
```